crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -906,6 +897,34 @@ impl AcirContext {
         }
     }
 
```

Comment:
can you put a doc comment here to state what this method is doing and why its needed?